pwa/src/templates/mappingDetailTemplate/MappingDetailsTemplate.tsx

```diff
@@ -50,7 +91,56 @@ export const MappingDetailTemplate: React.FC<MappingDetailsTemplateProps> = ({ m
             handleToggleDashboard={{ handleToggle: toggleFromDashboard, isActive: !!dashboardCard }}
           />
 
-          <MappingFormTemplate mapping={getMapping.data} />
+          <div className={styles.tabContainer}>
+            <TabContext value={currentTabs.mappingDetailTabs.toString()}>
+              <Tabs
+                value={currentTabs.mappingDetailTabs}
+                onChange={(_, newValue: number) => {
+                  setCurrentTabs({ ...currentTabs, mappingDetailTabs: newValue });
+                }}
+                variant="scrollable"
+              >
+                <Tab className={styles.tab} label={t("General")} value={0} />
+                <Tab className={styles.tab} label={t("Test Mappings")} value={1} />
+              </Tabs>
+
+              <TabPanel className={styles.tabPanel} value="0">
+                <MappingFormTemplate mapping={getMapping.data} />
+              </TabPanel>
+
+              <TabPanel className={styles.tabPanel} value="1">
+                <form onSubmit={handleSubmit(onSubmitTest)}>
+                  <Button
+                    className={clsx(styles.buttonIcon, styles.testConnectionButton)}
```

Comment:
I think `styles.testMappingButton` would be a more descriptive name here.

Also: we now have a `Button` component in the project, please merge the `refinement-demo-branch` and implement the new buttons before merging this 👍🏼